docs/settings/settings.md

```diff
@@ -27,9 +41,22 @@ Default: none
 }
 ```
 
+## `cluster-registration-url`
+
+This setting allows you to import the Harvester cluster to Rancher for multi-cluster management.
```

Comment:
```suggestion
This setting allows you to import the Harvester cluster to Rancher unified multi-cluster app management.
```

Reply:
rephrased based on the page https://rancher.com/why-rancher (Rancher unified multi-cluser app management)

Reply:
This is for managing multiple Harvester clusters using Rancher(https://docs.harvesterhci.io/v0.3/rancher/virtualization-management/#virtualization-management). App management is not the focus in Harvester use case.